src/app/heroes/heroes.component.ts

```diff
@@ -1,17 +1,24 @@
 import { Component, OnInit } from '@angular/core';
+import { Hero } from '../hero';
+import { HEROES } from '../mock-heroes';
 
 @Component({
   selector: 'app-heroes',
   templateUrl: './heroes.component.html',
   styleUrls: ['./heroes.component.css']
 })
+
 export class HeroesComponent implements OnInit {
 
-  hero = 'Windstorm';
+  heroes = HEROES;
+  selectedHero: Hero;
 
   constructor() { }
 
   ngOnInit() {
   }
 
+  onSelect(hero: Hero): void {
```

Comment:
Do you know what this function does?

Reply:
onSelect is called when new hero is selected and it just changes it to the new hero.